docs/design/managed_identity_capabilities_devex.md

```diff
@@ -0,0 +1,80 @@
+# Microsoft.Identity.Web – Claims & Client Capabilities Mini‑Spec (Managed Identity)
+
+## Why ClientCapabilities (cp1)?
```

Comment:
I would reorder this. This spec is not about client capabilities, it is about CAE.

Reply:
fixed